EIPS/eip-medgasprice_opcode.md

```diff
@@ -0,0 +1,93 @@
+---
+eip: <to be assigned>
+title: OPCODE 0x46 MEDGASPRICE
+author: Justice Hudson (@jchancehud)
+discussions-to: https://ethereum-magicians.org/t/medgasprice-opcode-eip/5480
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-03-05
+---
+
+## Simple Summary
+
+An opcode allowing smart contracts to access information about recent gas prices.
```

Comment:
I recommend referring to the opcode here using the format `MEDGASPRICE (0x46)` instead of saying "an opcode".